javatests/com/williamfiset/algorithms/math/isPrimeTest.java

```diff
@@ -0,0 +1,18 @@
+package javatests.com.williamfiset.algorithms.math;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.williamfiset.algorithms.math.ChineseRemainderTheorem;
+import com.williamfiset.algorithms.utils.TestUtils;
+import java.util.*;
+import org.junit.*;
+
+public class isPrimeTest {
+
+    @Test
+    public void testIsPrime() {
+      ChineseRemainderTheorem cRT = new ChineseRemainderTheorem();
+      boolean output = cRT.isPrime(21169);
```

Comment:
This isPrime should be from com/williamfiset/algorithms/math/IsPrime.java you'll also need to make the isPrime method public so that it's visible outside the package.